lib/order.rb

```diff
@@ -0,0 +1,32 @@
+class Order
+  attr_reader :selected_items
+  def initialize(menu_class = Menu, dish_class = Dish, confirmation_class = Confirmation)
+    @selected_items = []
+    @menu = menu_class.new([dish_class.new("chips", 1.50), dish_class.new("nuggets", 2.00),
+    dish_class.new("big mac", 3.50), dish_class.new("apple pie", 0.75)])
+    @total = 0
+    @confirmation_class = confirmation_class
+  end
+  def display_menu
+    @menu.dishes.each_with_index { |dish, index| puts "#{index + 1}. #{dish.name} £#{dish.price}"}
```

Comment:
Great idea to add an index to iterate through the dishes. Rather than try and use a hash.